### ISSUE TITLE
update of metrics database workflow

### ISSUE THREAD

**Issue by ehenneken** [adsabs/adsdata#61]
With the deprecation of ADS 2.0, we no longer need the MongoDB metrics_data collection. Removing this dependecy from the daily `build_docs` update will not only speed things up, it will also allow the metrics data pipeline to become more resilient.

Currently, if a record gets updated in the MongoDB collection, but the update of the Postgres database fails for some reason, the Postgres record will always stay out-of-date if the MongoDB record does not get updated.

**Comment by jonnybazookatone** [comment]
Yeah, ADSData needs a complete overhaul. This is one of the ideas for the new hire :smile:.

**Comment by adsabs-beer-user** [comment]
I'll do a small update so that at least the Postgres metrics data
collection stays sane and in sync

**Comment by aaccomazzi** [comment]
Before we kill the mongo collection please check and see what the impact on the metrics view generated in ADS classic is.  Does it break it completely?  And if so, can it be updated to use BBB?

**Comment by ehenneken** [comment]
ADS Classic does not use the MongoDB collection. It uses the ADS Classic index files. The module that takes care of everything is `ADSStats.py`. Personally I'd be very much in favor of disabling the metrics on Classic and to promote export of records to BBB and do metrics from there. The metrics on BBB have some extra information anyway.

**Comment by aaccomazzi** [comment]
Good to hear.  I don't want to spend a lot of effort on classic stuff but this might entice people over to BBB.  I'd still like to get the metrics for 3K records to work well in BBB before we remove the buttons though.
